README.md

```diff
@@ -77,6 +77,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Anime
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [AniDB](https://wiki.anidb.net/API) | Anime Database | No | Yes | Unknown |
```

Comment:
```suggestion
| [AniDB](https://wiki.anidb.net/HTTP_API_Definition) | Anime Database | `apiKey` | Yes | Unknown |
```

Reply:
I agree with the link change, but the authentication part is a little different. It is not a key, but a "client", an user-defined string that is required to be registered with an account, but handled like an user-agent.

Example: http://www.anidb.net:9001/httpapi?request=anime&client=animerank&clientver=1&protover=1&aid=1

I also just noticed that the API doesn't support HTTPS, my bad!

Shall we keep the authentication part with a `No`, have it be `apiKey` or is there another option?

Reply:
`apiKey` would be more appropriate as you need an account

Reply:
```suggestion
| [AniDB](https://wiki.anidb.net/HTTP_API_Definition) | Anime Database | `apiKey` | No | Unknown |
```